.github/workflows/develop.yml

```diff
@@ -518,6 +518,9 @@ jobs:
       - name: Setup kafka server
         working-directory: tests-integration/fixtures/kafka
         run: docker compose -f docker-compose-standalone.yml up -d --wait
+      - name: Setup minio
+        working-directory: tests-integration/fixtures/minio
+        run: docker compose -f docker-compose-standalone.yml up -d --wait
```

Comment:
- Is it a good idea to set up a minio in CI? Since we already have a S3 bucket for testing. When should we use minio?
- Should we modify our S3 tests to use minio?

Reply:
> When should we use minio?
When we need to test common behaviors in unit tests.

>  Should we modify our S3 tests to use minio?
No, It's fine to continue using S3 in integration tests